### PR TITLE
Allow to associate article from outing detail view page

### PR DESCRIPTION
**Pull request by dkocich**
Related to https://github.com/c2corg/v6_ui/issues/1284
It is now able to associate articles (and routes) from outing detail view page

### CODE REVIEW

File: c2corg_ui/static/js/addassociation.js [hunk 1]
```diff
@@ -88,7 +88,8 @@ app.AddAssociationController.prototype.associate = function(doc) {
   // if the parent doc is a route and the child doc is a waypoint OR
   // if the parent doc is an outing, inverse the IDs.
   if ((parentType === 'routes' && doc['type'] === 'w') ||
-      parentType === 'outings' || parentType === 'images' ||
+      (parentType === 'outings' && (doc['type'] === 'r' || doc['type'] === 'u')) ||
```
Comment (asaunier) [inline c2corg_ui/static/js/addassociation.js]: I am not sure to understand this change. Isn't article type missing here?

Comment (dkocich) [inline c2corg_ui/static/js/addassociation.js]: Outings were swapped to be child in all added associations on the outing view page (but in the case of articles we should keep them as children in its association to outings and there is no need to include them here - for outings only route/user should be swapped). https://github.com/c2corg/v6_common/blob/master/c2corg_common/associations.py#L7-L8

Comment (asaunier) [inline c2corg_ui/static/js/addassociation.js]: OK thanks for the explanations.

To be honest I think this part is getting a bit confusing and not easy to understand. I don't know if we could do something to make it more explicit?

**Comment by asaunier** [comment]
I have merged the PR because we would like to update the demo soon :)

**Pull request merged**